curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/specify-upper-and-lower-number-of-matches.english.md

```diff
@@ -53,9 +60,6 @@ let result = ohRegex.test(ohStr);
 ```
 
 </div>
-
-
-
```

Comment:
@ojeytonwilliams Correct me if I am wrong, but don't we need to have a blank line between the closing `div` and closing `section` tags based on the proposed linting in your PR?

Reply:
Nope, this is fine as far as the linter is concerned.  `prettier` would remove any empty lines between closing tags, but we're not going to apply it to the challenges.

Reply:
We were going to, but I kept finding more and more challenges it broke, so I'm reverting that change.

Reply:
So we're good to remove these lines then @ojeytonwilliams ?

Reply:
I checked with him before he signed off for the weekend and he says it is fine.